features/events/syslog/pom.xml

```diff
@@ -110,6 +110,12 @@
     <dependency>
       <groupId>org.opennms.core</groupId>
       <artifactId>org.opennms.core.daemon</artifactId>
+     <exclusions>
```

Comment:
This isn't right, you should leave this in. log4j-over-slf4j redirects log4j 1.x calls to slf4j which is what we want to do inside OpenNMS.